features/step_definitions/common_steps.rb

```diff
@@ -506,8 +506,9 @@
     page.find_by_id(
       page.all(
```

Comment:
Where's the closing bracket corresponding to this?

Reply:
```
page.find_by_id(
  page.all(
    :xpath,
    "//div[contains(@class, 'govuk-option-select')]//input[@type='checkbox']"
  ).sample["aria-controls"], visible: :all
).text(:all)
```
tl;dr in the middle of line 509

Reply:
Ahhhhhhhhhhh